tests/functionaltests/UXTestHelpers.mm

```diff
@@ -106,4 +114,27 @@
     }
 }
 
+//
+// UXEvent methods
+//
+void UXEvent::Signal() {
+    [_condition lock];
+
+    _signaled = true;
+    [_condition signal];
+
+    [_condition unlock];
+}
+
+bool UXEvent::Wait(int timeOutInSeconds) {
```

Comment:
hmm - i think Wait's more clear and aligns with most other event types.  We could return an enum vs a bool though for clarity, but true for succeeded, false for timed out also seems pretty clear?

---
In reply to: [98267209](https://github.com/Microsoft/WinObjC/pull/1855#discussion_r98267209) [](ancestors = 98267209)